lib/state/util/web3auth-config.ts

```diff
@@ -0,0 +1,67 @@
+import { BaseDotsamaWallet } from "@talismn/connect-wallets";
+import { CHAIN_NAMESPACES } from "@web3auth/base";
+import { Web3Auth } from "@web3auth/modal";
+import { atom } from "jotai";
+
+export const clientId = process.env.NEXT_PUBLIC_WEB3AUTH_CLIENT_ID_ZTG ?? "";
+
+class Web3AuthWallet extends BaseDotsamaWallet {
+  constructor({ extensionName, title, installUrl, logo }) {
+    super();
+    this.extensionName = extensionName;
+    this.title = title;
+    this.installUrl = installUrl;
+    this.logo = logo;
+    this.signer;
+  }
+}
+
+const extensionName = "web3auth";
+const title = "Web3Auth";
+const installUrl = "";
+const logoSrc = "/web3auth.svg";
+const logoAlt = "web 3 auth";
+
+export const web3AuthWalletInstance = new Web3AuthWallet({
+  extensionName,
+  title,
+  installUrl,
+  logo: {
+    src: logoSrc,
+    alt: logoAlt,
+  },
+});
+
+const chainConfig = {
+  chainNamespace: CHAIN_NAMESPACES.OTHER,
+  chainId: "0x1",
+  rpcTarget: "https://rpc.polkadot.io/",
+  displayName: "Polkadot Mainnet",
+  blockExplorer: "https://explorer.polkascan.io/",
```

Comment:
Not entirely sure what these configs do, and doesnt seem to be needed to sign transactions. But maybe they should be configured to zeitgeist?

Reply:
From my understanding it's used to generate the keyring pair for users. I'm not sure if it's compatible with Zeitgeist specific settings but I can try those instead and see if it works.